tutorials/CountAndCache.qmd

```diff
@@ -122,3 +123,221 @@ result as usual:
 ```{julia}
 get_solver_result(r)
 ```
+
+## Advanced Caching Examples
+
+There are more options other than caching single calls to specific parts of the objective.
+For example you may want to cache intermediate results
+of computing the cost and share that with the gradient computation. We will present
+three solutions to this:
+
+1. An easy approach from within `Manopt.jl`: The [`ManifoldCostGradientObjective`]
```

Comment:
You probably missed link address here?

Reply:
Actually just a `(@ref)` missing since it end up in the docs.